docs/integrations/saas-cloud/airtable.md

```diff
@@ -0,0 +1,100 @@
+---
+id: airtable
+title: Sumo logic App for Airtable
+sidebar_label: Airtable
+description: The Sumo Logic App for the Airtable App offers functionality for monitoring and analyzing your organization's Airtable audit logs.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/send-data/airtable-icon.png')} alt="airtable-icon.png" width="150" />
+
+The Airtable App for Sumo Logic monitors and analyses your organization's Airtable audit logs, providing insights into user activity, data trends, and security events. This app is based on the Airtable Audit Logs, which provide detailed information on all actions performed in your Airtable account.
```

Comment:
```suggestion
The Airtable App for Sumo Logic monitors and analyzes your organization's Airtable audit logs and provides insights into user activity, data trends, and security events. This app is based on the Airtable Audit Logs, which provide detailed information on all actions performed in your Airtable account.
```

Reply:
@kimsauce Oh! thank you "analyzes" needs to be fixed, but the second suggestion will not apply here. 
These two are independent clauses and when two independent clauses are related in meaning and are closely connected, a conjunction is not always necessary to join them. Instead, a comma can be used to separate the two clauses. This is known as a comma splice.